sw/device/tests/rv_core_ibex_rnd_test.S

```diff
@@ -0,0 +1,46 @@
+// Copyright lowRISC contributors.
```

Comment:
Any specific reason why these functions are specifically written in assembly, rather than reusing the DIFs?

Reply:
I've added a comment inside the assembly file explaining the reasoning here.

Reply:
Thanks!